frontend/src/metabase/visualizations/visualizations/PivotTable.jsx

```diff
@@ -363,48 +356,41 @@ export default class PivotTable extends Component {
                   {rowIndexes.map(index => (
                     <Cell
                       value={formatColumn(columns[index])}
-                      baseWidth={LEFT_HEADER_CELL_WIDTH}
-                      width={1}
-                      height={1}
+                      style={{ width: LEFT_HEADER_CELL_WIDTH }}
                     />
                   ))}
                 </div>
                 {/* top header */}
-                <Grid
-                  ref={e => (this.topGrid = e)}
+                <Collection
                   className="scroll-hide-all text-medium"
                   width={width - leftHeaderWidth}
                   height={topHeaderHeight}
-                  rowCount={1}
-                  rowHeight={topHeaderHeight}
-                  columnCount={columnCount}
-                  columnWidth={columnWidth}
-                  cellRenderer={topHeaderRenderer}
+                  cellCount={topHeaderItems.length}
+                  cellRenderer={topCellRenderer}
```

Comment:
Just for my understanding, does the name `topCellRenderer` here convey something special?
I have the impression that the previous name `topHeaderRenderer` is a little less ambiguous (and grouped nicely with other `topHeader-`, great for code search etc).

Reply:
Good point. Maybe `topHeaderCellRenderer` is the best description. It doesn't render the whole header, but the cell it renders is specifically in the header.